server/manager/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1422,7 +1423,8 @@ private void getMasterLock(final String zMasterLoc) throws KeeperException, Inte
     while (true) {
 
       MasterLockWatcher masterLockWatcher = new MasterLockWatcher();
-      masterLock = new ZooLock(context.getZooReaderWriter(), zMasterLoc);
+      masterLock =
+          new ZooLock(context.getZooReaderWriter(), zMasterLoc, ZooReader.DISABLED_RETRY_FACTORY);
```

Comment:
In these kinds of uses, the code reads as though ZooLock itself won't retry... this is a bit unintuitive, because of the unfortunate name.